packages/flutter/test/material/text_field_test.dart

```diff
@@ -489,7 +489,7 @@ void main() {
     await tester.tap(find.byKey(icon));
     await tester.pump();
     expect(controller.text, '');
-    expect(controller.selection, const TextSelection.collapsed(offset: 0, affinity: TextAffinity.upstream));
```

Comment:
Why this change?

Reply:
I think before the change the tap event actually altered the selection. Now it's only hitting the button and not the text field so that should be expected.